server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java

```diff
@@ -1097,6 +1082,18 @@ private static String pathFromRevision(long revision) {
         return String.format("%010d", revision);
     }
 
+    private <T> ReplicationTimings newReplicationTimings(Command<T> command) {
+        ReplicationMetrics metrics = null;
+        if (command instanceof RepositoryCommand) {
```

Comment:
nit; If only `RepositoryCommand`s are recorded, I don't see a reason not to also record per repository. As is seems fine though.

Reply:
We used to record metrics per repository in `MetricCollectingService`, but it caused excessive memory usage and frequent GC, so I took a more conservative approach. If we later find the project-level metrics are insufficient, we may consider tagging repositories.